feg/gateway/Makefile

```diff
@@ -52,8 +53,19 @@ gen:
 vet:
 	go vet ./...
 
-lint:
-	golint -min_confidence 1. ./...
+lint: lint_tools
+	golangci-lint run
+
+lint_tools:
+	#if which golangci-lint 2>/dev/null; then echo "-> golangci-lint already installed"; \
+	#else curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh \
+	#		| sh -s -- -b $$(go env GOBIN) v1.35.2; \
+	#fi
```

Comment:
Remove extraneous comments (?)

Reply:
Yep, this is still under test. I am switching back and forth. I will  remove it once I am done.